src/components/structures/RoomView.js

```diff
@@ -748,6 +754,21 @@ export default createReactClass({
             }
         }
     },
+    onEventDecrypted(ev) {
+    if (!SettingsStore.getValue('dontShowChatEffects')) {
+        if (ev.isBeingDecrypted() || ev.isDecryptionFailure() ||
```

Comment:
these checks will always be false given that you are are fired from the `Event.decrypted` event